CMakeLists.txt

```diff
@@ -139,8 +139,13 @@ elseif (APPLE AND CMAKE_SYSTEM_PROCESSOR STREQUAL "arm64")
   set(${P}_NATIVE_FLAGS_DEFAULT "-mcpu=apple-m1")
 else()
   if (SKBUILD)
-    # Reasonably portable binaries for PyPI
-    set(${P}_NATIVE_FLAGS_DEFAULT "-march=ivybridge")
+    if (CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|arm64|ARM64")
+      # ARM64 architecture - use native or a baseline ARM arch
+      set(${P}_NATIVE_FLAGS_DEFAULT "-march=armv8-a")
```

Comment:
I don't have an opinion about which "baseline" arch to target, we can change that.